### PR TITLE
[RV64_DYNAREC] Added more opcodes and some fixes

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

File: src/dynarec/rv64/dynarec_rv64_660f.c [hunk 1]
```diff
@@ -245,29 +303,43 @@ uintptr_t dynarec64_660F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
...
+                    BNEZ_MARK(x2);
+                    FADDD(v0, d0, d0);
+                    B_NEXT_nocond;
+                    MARK; // d0 is not nan
+                    FABSD(v0, d0);
+                    MOV64x(x3, 1ULL << __DBL_MANT_DIG__);
+                    FCVTDL(d1, x3, RD_RTZ);
+                    FLTD(x3, v0, d1);
+                    BNEZ_MARK2(x3);
+                    FMVD(v0, d0);
```
Comment (ptitSeb) [inline src/dynarec/rv64/dynarec_rv64_660f.c:326]: you should test if v0!=d0

Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_660f.c:326]: You mean something like `if (v0!=d0) FMVD(v0, d0);`?

File: src/dynarec/rv64/dynarec_rv64_660f.c [hunk 2]
```diff
@@ -1028,6 +1100,30 @@ uintptr_t dynarec64_660F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
...
+                // GX->sb[i] = (tmp16s>127)?127:((tmp16s<-128)?-128:tmp16s);
+                LB(x3, gback, i);
+                LB(x4, wback, fixedaddress+i);
+                ADDW(x3, x3, x4);
+                SLLIW(x3, x3, 16);
+                SRAIW(x3, x3, 16);
+                ADDI(x4, xZR, 0x7f);
+                BLT(x3, x4, 12);     // tmp16s>127?
+                SB(x4, gback, i);
+                BEQ(xZR, xZR, 24);   // continue
```
Comment (ptitSeb) [inline src/dynarec/rv64/dynarec_rv64_660f.c:1119]: probably not

File: src/dynarec/rv64/dynarec_rv64_660f.c (hunk 1 above)
Comment (ptitSeb) [reply src/dynarec/rv64/dynarec_rv64_660f.c:326]: Yes, no need to move the reg if it's same

Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_660f.c]: Done

File: src/dynarec/rv64/dynarec_rv64_660f.c (hunk 2 above)
Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_660f.c]: Yup, `jal x0` is better, done.

**Pull request merged**